frame/mosaic/src/lib.rs

```diff
@@ -714,6 +763,44 @@ pub mod pallet {
 			}
 			Ok(().into())
 		}
+
+		// TODO: Set proper pallet weight
```

Comment:
is mosaic pallet on picasso? if yes, then when it will be released or what is task to have weight here so it is done before release?

Reply:
Oh oops I forgot to adjust the weights. Will fix